### PR TITLE
fix(web): wrap long album title

### LINKED ISSUE #26533: [Web] [Issue] Web UI does not properly display long album names
### I have searched the existing issues, both open and closed, to make sure this is not a duplicate report.

- [x] Yes

### The bug

When the album name is very long, it is cut off on the web app and it is not fully visible, cause the text size is too big, it does not fit on one screen width. 

See this image on demo server (I noticed this issue on my server but then I tested demo server to see if i can reproduce the same)

If i put a long Album name say `/Volumes/990_Pro/Leo/EXPORTED-DATA/Google Takeout/myfakemeail@gmail.com/GDRIVE/25-02-2026`

It gets cut off at the end, and there is no scroll bar either to see the full name:

Image

The expected behaviour is if the album name is longer than certain length, shrink down the font size so it can fit one window width. 

### The OS that Immich Server is running on

Irrelevant [can be reproduced on demo server]

### Version of Immich Server

v2.5.6

### Version of Immich Mobile App

NA

### Platform with the issue

- [ ] Server
- [x] Web
- [ ] Mobile

### Device make and model

Irrelevant [This is a web related issue]

### Your docker-compose.yml content

```YAML
Irrelevant [can be reproduced on demo server]
```

### Your .env content

```Shell
Irrelevant [can be reproduced on demo server]
```

### Reproduction steps

1. Go to demo server
2. Create an album with a relatively long name
3. See the name cut off on the right, se this test album I created, or create one yourself [Test Album](https://demo.immich.app/albums/043b1102-5d61-43eb-8083-abf1fe302d8b)

Even though it's an edge case, it should be fixed ...

### Relevant log output

```shell
Irrelevant [can be reproduced on demo server]
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jrasm91**
Wrap album title

Fixes #26533

### CODE REVIEW

File: web/src/lib/components/album-page/album-description.svelte [hunk 1]
```diff
@@ -33,10 +33,7 @@
 {#if isOwned}
   <Textarea
     bind:value={description}
-    class="outline-none border-b max-h-32 border-transparent pl-0 bg-transparent ring-0 focus:ring-0 resize-none focus:border-b-2 focus:border-immich-primary dark:focus:border-immich-dark-primary dark:bg-transparent"
```
Comment (michelheusschen) [inline web/src/lib/components/album-page/album-description.svelte:36]: The `max-h-32` class limited the description height. I think it makes sense to keep a height limit and we should probably add that to the title as well.

File: web/src/lib/components/album-page/album-title.svelte [hunk 2]
```diff
@@ -3,59 +3,54 @@
...
-  use:shortcut={{ shortcut: { key: 'Enter' }, onShortcut: (e) => e.currentTarget.blur() }}
-  onblur={handleUpdateName}
-  class={styles({ isOwned })}
-  type="text"
-  bind:value={newAlbumName}
-  disabled={!isOwned}
-  title={$t('edit_title')}
-  placeholder={$t('add_a_title')}
-/>
+<div class="p-2 mb-2">
```
Comment (michelheusschen) [inline web/src/lib/components/album-page/album-title.svelte]: The spacing looks a bit off now. It might be better to keep the spacing between the title and description consistent.

| Before | This PR |
| - | - |
| Image | Image |

File: web/src/lib/components/album-page/album-title.svelte [hunk 3]
```diff
@@ -3,59 +3,54 @@
...
+  {#if isOwned}
+    <Textarea
+      bind:value={newAlbumName}
+      variant="ghost"
+      title={$t('edit_title')}
```
Comment (michelheusschen) [inline web/src/lib/components/album-page/album-title.svelte:46]: Now the title has become a textarea it can also contain newlines. Mobile doesn't seem to handle that well. Editing a title on mobile will remove those newlines and the UI does not display multiline titles properly.

Comment (jrasm91) [reply web/src/lib/components/album-page/album-title.svelte:46]: How do you think this should be addressed? I think new lines should probably be stripped. When the user presses the `Enter` key it submits the form instead of adding a new line, right?

Comment (michelheusschen) [reply web/src/lib/components/album-page/album-title.svelte:46]: Yeah removing newlines is probably best, you can add newlines using `Shift + Enter`

File: web/src/lib/components/album-page/album-description.svelte (hunk 1 above)
Comment (jrasm91) [reply web/src/lib/components/album-page/album-description.svelte:36]: I added it back to the description. I think the title is fine without it though. There's no restriction on google photos and I think it looks fine.

**Comment by jrasm91** [comment]
@michelheusschen I think I've fixed all those issues. Do you mind checking again?

**Review by michelheusschen — APPROVED** [review]
Looks good, thank you 👍

**Review by danieldietzler — APPROVED** [review]
